packages/core-components/src/components/OAuthRequestDialog/OAuthRequestDialog.tsx

```diff
@@ -83,6 +94,7 @@ export function OAuthRequestDialog(_props: {}) {
           <Typography className={classes.titleHeading} variant="h1">
             Login Required
           </Typography>
+          <Typography>{redirectMessage}</Typography>
```

Comment:
Let's avoid rendering this altogether if `redirectMessage` isn't set. Always nice to keep the DOM clean for screen readers etc.